### ISSUE TITLE
collapsible member grouping

### ISSUE THREAD

**Issue by doggybootsy** [powercord-community/suggestions#388]
### Description
makes the member group act like channels with the ability to collapse 

### More info
Before: https://i.imgur.com/LvqYPVV.png
After: https://i.imgur.com/6ZXRsci.png

----

 - [x] I checked if this request wasn't already proposed
 - [x] I made sure this request follows Powercord's [plugin guidelines](https://powercord.dev/guidelines)

**Comment by GriefMoDz** [comment]
Duplicate of #16

**Comment by doggybootsy** [comment]
oh thx
